tasks/main.yml

```diff
@@ -18,3 +18,17 @@
     name: "{{ item }}"
     global: yes
   with_items: npm_packages
+
+- name: Set explicit versions of node and npm
+  command:
+    name: "/usr/bin/npm cache clean -f"
+  command: 
+    name: "/usr/bin/npm install -g n"
+  command:
+    name: "/usr/bin/n {{ __node_version }}"
+  command:
+    name: "/bin/ln -sf /usr/local/n/versions/node/{{ __node_version }}/bin/node /usr/local/bin/node"
```

Comment:
I suggest using `alternatives` for manage the symbol link

http://docs.ansible.com/ansible/alternatives_module.html